cmd/cp-main_test.go

```diff
@@ -28,11 +28,11 @@ func TestParseMetaData(t *testing.T) {
 		err    error
 		status bool
 	}{
-		// success scenerio using ; as delimitter
+		// success scenario using ; as delimitter
```

Comment:
please fix typo delimitter -> delimiter wherever they occur

Reply:
thanks for pointing it out. Done with the change.